### DISCUSSION TITLE
Log back-end location

### DISCUSSION THREAD

**Discussion by karasz** [darvaza-proxy/slog#15] [Ideas]
I was thinking if we should not have a backends folder for the log backends. Having them all in root seems a bit messy.
Something like 
```bash
mkdir -p backends
git mv zerolog backends/zerolog
```

It will make #12 a bit easier to implement.

**Comment by amery** [comment]
not sure backend is the right term.

I was wondering about movings tools into internal so there is only one special directory

**Reply by karasz** [reply]
I think "driver" is worse. Any proposal?

**Reply by karasz** [reply]
"loggers"?

**Reply by amery** [reply]
backends seems like the least bad option 🤷🏻‍♂️

**Comment by amery** [comment]
> It will make #12 a bit easier to implement.

why?

**Reply by karasz** [reply]
Because you can write file tests *_test.go and backend_test.go which deals with "compliance"

**Reply by amery** [reply]
please send the PR and we can merge it right away

**Comment by amery** [comment]
as of slog v0.2.0 they are in a `handlers/` subdirectory
